.github/workflows/pr_badges.yaml

```diff
@@ -0,0 +1,57 @@
+name: PR badges
+
+on:
+  pull_request:
+    branches: [main]
+
+concurrency:
+  group: '${{ github.workflow }}-${{ inputs.platform }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.sha }}'
+  cancel-in-progress: true
+
+permissions:
+  pull-requests: write
```

Comment:
Why do we need this permission?  How is it going to affected workflows from forked PRs, those are pegged to read-only permissions?

Reply:
Yes, the bot needs this permission to be able to post a comment on the PR. I can test the forks flow in my repo to see how they are going to react to it.

Reply:
I tried to create PR from my fork and it failed, see https://github.com/youtube/cobalt_sandbox/actions/runs/3964022328/jobs/6792501730

What we need to do is use pull_request_target context instead.